docs/maps/asset-tracking-tutorial.asciidoc

```diff
@@ -183,6 +183,7 @@ image::maps/images/asset-tracking-tutorial/discover.png[]
 
 [float]
 === Part 2: Build an operational map
+Its hard to get an overview of Portland buses by looking at individual events. Lets create a map to show the bus routes and current location for each bus along with the direction buses are headed.
```

Comment:
It's hard to get an overview of Portland buses by looking at individual events. Let's create a map to show the bus routes and current location for each bus, along with the direction the buses are headed.